### DISCUSSION TITLE
How to run AWICM3-v3.1 with annual restart frequency?

### DISCUSSION THREAD

**Discussion by mbutzin** [esm-tools/esm_tools#810] [ESM-Tools support]
To my understanding this could be controlled by setting "restart_unit" to "years" in both fesom and oifs sections of the awicm3 runscript. However, then the model execution slows down and is finally cancelled, even if the time limit is set to 8 hours (at Levante). What am I missing?

**Comment by JanStreffing** [comment]
Hello Martin,
are you running with 
```
oasis:
     lresume: true
```
or is it set to false? If it is set to false, we expect a slowdown, because the model is asked to generate oasis restart files (rstos.nc and rstas.nc) from oasis debug output on the first leg. My recommendation is to have the first leg be only one day long. You can then store the restart files and link them in. The model will not slow down when run in lresume:true mode.

**Comment by mandresm** [comment]
@mbutzin there is no need for a `oifs.restart_unit` as the OpenIFS is configured different than other models and the restarts are configured to happen at the end of each run.

About the performance, is this still an issue or did the change suggested by Jan helped?
